src/main/resources/io/fabric8/elasticsearch/plugin/kibana/empty_project_mappings.json

```diff
@@ -0,0 +1,6 @@
+{
+   "title":"$TITLE$",
+   "timeFieldName":"time",
+   "description":"This is the Kibana index file for empty projects",
+   "fields": "[{\"name\":\"_index\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"indexed\":false,\"analyzed\":false,\"doc_values\":false},        {\"name\":\"_type\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"indexed\":true,\"analyzed\":false,\"doc_values\":false},{\"name\":\"_source\",\"type\":\"_source\",\"count\":0,\"scripted\":false,\"indexed\":false,\"analyzed\":false,\"doc_values\":false},{\"name\":\"_id\",\"type\":\"string\",\"count\":0,\"scripted\":false,\"indexed\":false,\"analyzed\":false,\"doc_values\":false},{\"name\":\"time\",\"type\":\"date\",\"count\":0,\"scripted\":false,\"indexed\":true,\"analyzed\":false,\"doc_values\":false}\r\n    ]"
```

Comment:
Tried single quotes and Kibana doesn't seem to want to parse it.  I'm leaving for now.